docs/source/usage/cleaning.rst

```diff
@@ -0,0 +1,189 @@
+  2. Erase Devices Metadata: Wipe partition tables, signatures, file-system identifications, and other disk devices.
+  3. RAID Delete (Disabled-by-default): Delete any existing RAID configuration. This step needs to be enabled for automated cleaning.
+  4. Setup RAID configuration in advance(Disabled-by-default): Create RAID configuration from a predefined template when the node in enrolled. This step needs to be enabled for automated cleaning.
+
+
+Enable or Disable Automated Cleaning
+------------------------------------
+To enable automated cleaning, ensure that your ironic.conf has following option enabled.
+
+.. prompt:: bash $
+
+  [conductor] ironic.conf
+  automated_clean=true
+
+To disable it, ensure that your ironic.conf has following option disabled.
+
+.. prompt:: bash $
+
+  [conductor] ironic.conf
+  automated_clean=false
+
+Configuring Automated Cleaning Steps
+------------------------------------
+
+A cleaning step can be disabled by setting the priority for that cleaning step to zero or ‘None’. For instance, to disable erase_devices, you would set the following configuration option:
+
+.. prompt:: bash $
+
+  [deploy]
+  erase_devices_priority=0
+
+
+A cleaning step can enabled and/or prioritized by setting a value for its priority. For example, Ironic operator can enable factory_reset step by specifying priority in ironic.conf:
+
+.. prompt:: bash $
+
+  [conductor]
+  clean_step_priority_override=bios.factory_reset:10
+
+
+Automatic cleaning configuration default priorities :
+  * erase_devices:10
+  * erase_devices_metadata:99
+
+
+More information about changing cleaning step priorities `here`_.
+
+Use Automated Cleaning
+----------------------
+
+Trigger Automated Cleaning:
+  Put the node in “manage” mode and then put it again in “deploy” mode to launch automated cleaning. Run the following commands:
+
+.. prompt:: bash $
+
+  openstack baremetal node manage <node>
+  openstack baremetal node deploy <node>
+
+
+Manual Node Cleaning
+~~~~~~~~~~~~~~~~~~~~
+
+`Manual cleaning`_ is a way to run driver-specific actions on manageable nodes. The mechanism is the same as with automated cleaning but it is not limited to just cleaning nodes. manual cleaning is typically used to handle long running, manual, or destructive tasks that an operator wishes to perform either before the first workload has been assigned to a node or between workloads.
+
+
+Enable Manual Cleaning
+----------------------
+
+Manual cleaning can only be performed when a node is in the manageable state, and NOT under maintenance mode. To put a node in the manageable state run the following command:
+
+.. prompt:: bash $
+
+    openstack baremetal node manage <node>
+
+Use Manual Cleaning
+-------------------
+
+Once a node is in the manageable state, we need to specify three parameters:
+
+    1. An interface in the cleaning step: management, raid, bios.
+    2. The desired `cleaning step`_ for the selected interface.
```

Comment:
A small thing, but I'd call out this link a lot more because it's super important. Maybe remove the link here, and put it up above, with something like:

Once a node is in the manageable state, it is ready for manual cleaning. The list of available manual cleaning steps can be found here (make this a link). Each step requires three parameters.

Reply:
Done!